src/app/components/secondary-navbar/secondary-navbar.component.html

```diff
@@ -1 +1,48 @@
-<p>secondary-navbar works!</p>
+<div class="secondary-navbar" *ngIf="isDesktop">
+    <div class="item about-us">About Us</div>
+    <div class="item collections" #collectionsAnchor (click)="dropdowns.toggleCollectionDropdown()">Collections
+        <span *ngIf="!collectionsDropdown">
+            <i class="fa fa-chevron-down"></i>
+        </span>
+        <span *ngIf="collectionsDropdown">
+            <i class="fa fa-chevron-up"></i>
+        </span>
+    </div>
+    <clark-context-menu *ngIf="collectionsDropdown" [anchor]="collectionsAnchor" (close)="dropdowns.closeAll()">
+        <div #contextMenu>
+            <ul>
+                <li>test</li>
+            </ul>
+        </div>
```

Comment:
Would this be a list of collections? Should this be hardcoded? Or should it use ngFor?

Reply:
there is a collections component that has yet to be created, I'm hoping I can just use that. But we'll see